src/testdrive/src/action/sql.rs

```diff
@@ -190,6 +191,11 @@ impl Action for SqlAction {
                 | Statement::CreateView { .. }
                 | Statement::DropDatabase { .. }
                 | Statement::DropObjects { .. } => {
+                    let temp_dir = tempfile::tempdir()?;
+                    let mut options = CopyOptions::new();
+                    options.content_only = true;
+                    fs_extra::dir::copy(&path, temp_dir.path(), &options)?;
+                    let path = temp_dir.path();
```

Comment:
Can you add a comment about why we do this?

Reply:
I extracted this out to it's own function/mod and put a comment there.